cloud.foundry.cli/src/main/java/cloud/foundry/cli/crosscutting/mapping/beans/ApplicationBean.java

```diff
@@ -24,6 +24,17 @@
     private String path;
     private String meta;
 
+    public ApplicationBean(ApplicationManifest manifest, Metadata meta) {
```

Comment:
The documentation should be added here (because this constructor differs from a default constructor)